Scrips.Core/Models/Patient/ProviderInsuranceModel.cs

```diff
@@ -18,6 +18,6 @@ public ProviderInsuranceModel()
         public string Email { get; set; }
         public string PhoneNumber { get; set; }
         public string Note { get; set; }
-        public List<ThirdPartyAdministrator> RelatedTpa { get; set; } = new();
+        public List<ThirdPartyAdministrator> RelatedTpa { get; set; }
```

Comment:
why did u remove this ?

Reply:
Added in constructor 
RelatedTpa = new List<ThirdPartyAdministrator>();